### ISSUE TITLE
Enable lambda functions on sidebar links 

### ISSUE THREAD

**Issue by iagobalmeida** [unfoldadmin/django-unfold#170]
Is there any possible way of implemeting a link that uses a lambda function?

I have a `Organization` Model, each `User` is associated with an `Organization`.

I want to show a link in the sidebar to the `change_form` view of the current user's `Organization` Model.

The ideal solution would be something like this:
 ```python
 #settings.py
 
 ...
 "SIDEBAR": {
        "show_search": False,  # Search in applications and models names
        "show_all_applications": False,  # Dropdown with all applications and models
        "navigation": [
        ...
        {
                "title": _("Organizações"),
                "separator": True,  # Top border
                "items": [
                    {
                        "title": _("Organizações"),
                        "icon": "corporate_fare",
                        "link": reverse_lazy("admin:organizations_organization_changelist"),
                    },
                    {
                        "title": _("Minha Organização"),
                        "icon": "corporate_fare",
                        "link": lambda request: get_user_organization(request.user).url
                    }
                ],
            },
        ],
    }
}
...
```

**Comment by DoubleTroubleOf** [comment]
@iagobalmeida 
As I understand, you should be able to register custom django view with is_staff or any other permission. and just return HttpResponseRedirect with you target URL.
It's not ideal solution. but as a walkaround shoudl work for you.

**Comment by lukasvinclav** [comment]
@iagobalmeida @DoubleTroubleOf implemented in #245
